index.html

```diff
@@ -361,3 +367,47 @@ <h1>
     <script src="index.js"></script>
   </body>
 </html>
+
+<!-- modal outlook -->
+<!-- <section class="modal">
+  <div class="modal-top">
+    <div class="project-title">
+      <h4>Keeping track of hundreds of components website</h4>
+    </div>
+    <div class="project-tools">
+      <ul>
+        <li>HTML</li>
+
+        <li>Bootstrap</li>
+
+        <li>Ruby</li>
+      </ul>
+    </div>
+  </div>
+  <div class="modal-bottom" id="modal-bottom">
+    <div class="project-img">
+      <img src="SVG/modalimgpng.png" alt=" Image" />
+    </div>
+    <div class="project-details">
+      <div class="project-desc">
+        <p>
+          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Expedita
+          laudantium eum animi nihil, recusandae blanditiis ut deleniti
+          perferendis veniam rem libero ea porro numquam culpa! Mollitia, ea
+          quisquam reprehenderit quia nam, temporibus repellendus et, rem
+          doloribus laboriosam laudantium. Aut, incidunt! Ab nisi ad repellendus
+          aperiam optio vero rerum esse minima?
+        </p>
+      </div>
+
+      <div class="project-links">
+        <a href="https://karanj2212.github.io/" id="project-link-1"
+          >See Live <i class="fas fa-external-link" aria-hidden="true"></i
+        ></a>
+        <a href="https://github.com/karanJ2212/portfolio" id="project-link-2"
+          >See Source <i class="fab fa-github" aria-hidden="true"></i
+        ></a>
+      </div>
+    </div>
+  </div>
+</section> -->
```

Comment:
- [ ] Kindly remove all commented code from your project, you can leave comments about your code but not commented code, this will make your codebase look neat and readable 😉

Reply:
Done. Thank you